packages/studio/frontend/src/views/FlowEditor/inspector/layout/TabGroup/Tab.tsx

```diff
@@ -0,0 +1,26 @@
+import { Icon } from '@blueprintjs/core'
+import React, { FC } from 'react'
+
+import style from './style.scss'
+
+export type TabOnClick = (id: string, idx: number, event: React.MouseEvent) => void
+
+interface OwnProps {
+  id: string
+  label: string
+  idx?: number
+  onClick?: TabOnClick
+  onDelete?: any
```

Comment:
Can you update the typing here.

Reply:
same thing as above, they are any for now, they will be updated with the next ticket.